app/controllers/titles_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class TitlesController < ApplicationController
-  before_action :set_title, only: %i[show customer_resources]
+  before_action :set_title, only: %i[show resources]
```

Comment:
As a general note (and completely out of scope of this PR), `before_action`s are overkill here. A simple method is just fine:

```ruby
def title
  titles.find params[:id]
end
```

Reply:
And since only the `show` and `resources` actions will every call it, the `only: ` clause enforced simply via usage.